recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  skip: true  # [osx]
```

Comment:
Most of these package can be built on Windows since they are dependent on packages that are available on Windows or are outputs of this feedstock. To get some of these packages out on Windows, drop this skip to enable those packages

However ones that still can't build on Windows (due to missing dependencies) are skipped individually below